addons/main/XEH_postInit.sqf

```diff
@@ -84,6 +84,14 @@ if (GVAR(aceMedicalLoaded)) then {
         (_this select 0) setDamage 0;
         _this call FUNC(handleHealEh);
     }] call CBA_fnc_addEventHandler;
+    
+    [QGVAR(consumeMedkit), {
+        hint "Consuming medkit";
```

Comment:
only show hint when FirdAidKit is actually consumed

Reply:
oops forgot to delete the hint lol